test/integration/cloud-config-aws.yml.template

```diff
@@ -16,6 +16,7 @@ aws_secret_key: @SECRET_KEY
 security_token: @SECURITY_TOKEN
 aws_region: @REGION
 # aliases for backwards compatibility with older integration test playbooks
+ec2_region: '{{ aws_region }}'
```

Comment:
This is redundant with the last line of the file.

Reply:
It is now - you only added that last line on Friday!

Reply:
I've removed my commit duplicating your recent work on this :)